db/corruption_test.cc

```diff
@@ -616,7 +616,7 @@ TEST_F(CorruptionTest, ParanoidFileChecksOnFlush) {
     options.table_factory = mock;
     mock->SetCorruptionMode(mode);
     ASSERT_OK(DB::Open(options, dbname_, &db_));
-    assert(db_ != nullptr);
```

Comment:
These asserts were put there for clang analyzer (without an informative comment 😢 ) and changing them leads to return of the false reports.